### ISSUE TITLE
[Diffie Hellman] Add description that a**b means a to the power of b?

### ISSUE THREAD

**Issue by Gautzilla** [exercism/problem-specifications#1932]
Hey,
I had some trouble with the Diffie Hellman exercise because of the instruction:

`A = g**a mod p`

Where `g**a` represents `g` to the power of `a`.

I thought it was basically a `a*b` and then struggled correcting my code. Not having a CS background, I was only used to the `a^b` notation.

Wouldn't it be useful to add a comment specifying that `a**b` means `a^b` or a to the power of b?

**Comment by SaschaMann** [comment]
Imo we should probably just change it to `^` or `aᵇ` in line with other exercises. Preferably the latter.

**Comment by ErikSchierboom** [comment]
Agreed. I found the `**` unintuitive too.

**Comment by SaschaMann** [comment]
@Gautzilla do you want to open a PR that changes it?

**Comment by Gautzilla** [comment]
> @Gautzilla do you want to open a PR that changes it?

Yes, I'll do that! (I'm still learning to use git, I hope I'll do it right!)

**Comment by ErikSchierboom** [comment]
@Gautzilla We have some useful links on how to create PRs here: https://exercism.org/docs/building

**Comment by Gautzilla** [comment]
> @Gautzilla We have some useful links on how to create PRs here: https://exercism.org/docs/building

Thank you, I was reading all that! 
I'm learning so much different things here it's insane. Thanks for making all this possible! ;)

**Comment by Gautzilla** [comment]
#1933 

I created a PR addressing this issue.
